### DISCUSSION TITLE
Please clarify relationship between ProcessHacker-2 and System Informer

### DISCUSSION THREAD

**Discussion by sydbarrett74** [winsiderss/systeminformer#1577] [Questions / Answers] [answered]
There has been some confusion with the various forks from the original Process Hacker. Could someone clarify the relationship (if any) between System Informer and [ProcessHacker-2](https://github.com/fengjixuchui/ProcessHacker-2)?

**Comment by Biswa96** [comment, marked as answer]
tl;dr This repository is the official one and others aren't.

**Comment by dmex** [comment]
> There has been some confusion with the various forks from the original Process Hacker.

* Process Hacker was renamed System Informer.
* The github repository and ownership was transferred to the winsiderss organization.
* The original repository automatically redirects to the new location:
https://github.com/ProcessHacker/ProcessHacker --> redirects --> https://github.com/winsiderss/systeminformer

The original Process Hacker repository will always redirect here.

> Could someone clarify the relationship (if any) between System Informer and [ProcessHacker-2](https://github.com/fengjixuchui/ProcessHacker-2)?

None. That project is an unofficial fork and not affiliated with Process Hacker or System Informer. Also the "commits ahead" for that repository is fake because of how they're using pull requests to merge changes.

**Reply by gggirlgeek** [reply]
All of the links in [ProcessHacker-2](https://github.com/fengjixuchui/ProcessHacker-2) go to https://processhacker.sourceforge.io/ and then to https://processhacker.sourceforge.io/nightly.php

When you download the file labeled "processhacker-3.0.5553-bin.zip" it downloads as "systeminformer-3.0.5553-bin.zip" When I open the zip, the Readme is for SystemInformer. The only thing that's weird is that SystemInformer's current nightly version is 3.0.5917.

The only reason I started using SI Nightly instead of PH2 Nightly was because I read somewhere on SI's github that the only difference now is that SI's certificate is signed, so the Antivirus programs leave it alone. And, indeed, it's true. My husband was having fits with PH when gaming too. I switched him to SI and now the games (on Steam) leave him alone.

So ProcessHacker-2 is related, obviously. It says on their github that the page is a mirror of SystemInformer's page.

Someone please explain.

**Reply by ge0rdi** [reply]
[ProcessHacker-2](https://github.com/fengjixuchui/ProcessHacker-2) is clone of [ProcessHacker](https://github.com/PKRoma/ProcessHacker) that is repository that someone _copied_ from original [ProcessHacker](https://github.com/ProcessHacker/ProcessHacker) repository.
This repo is not official one and never was, please, stay away of it.

[SystemInformer](https://github.com/winsiderss/systeminformer) is the official repository for `System Informer` (previously known as `Process Hacker`).

Install latest official `System Informer` [nightly build](https://systeminformer.sourceforge.io/nightly.php) and then just update it using built-in updater (`Help -> Check for updates`).
That way you will always have latest and official version of `System Informer`.

**Reply by dmex** [reply]
> When you download the file labeled "processhacker-3.0.5553-bin.zip" it downloads as "systeminformer-3.0.5553-bin.zip" When I open the zip, the Readme is for SystemInformer. The only thing that's weird is that SystemInformer's current nightly version is 3.0.5917.

The difference in version numbers was a synchronization bug which has now been fixed👍 

> All of the links in ProcessHacker-2 go to https://processhacker.sourceforge.io/

Anyone can link to the website. It doesn't make them official or legitimate.

> It says on their github that the page is a mirror

You can click the `Fork` button at the top right of Github nd clone the repository into your own personal account. There are currently 1.2k forks/clones of Process Hacker:
![image](https://user-images.githubusercontent.com/1306177/215736215-6c6d4b01-8855-44e2-86c5-59b2aa9ae1cc.png)

> So ProcessHacker-2 is related, obviously. It says on their github that the page is a mirror

No. They are not related. Process Hacker was renamed System Informer. Process Hacker does not exist anymore. The original repository (https://github.com/processhacker/processhacker) permanently redirects here. The old website (https://processhacker.sourceforge.io/) is being decommissioned and redirects here https://systeminformer.sourceforge.io/

**Comment by Ou7law007** [comment]
Is this related to the driver certificate problem? Is there a way to run the normal process hack 2 with the driver?
